docs/content/index.md

```diff
@@ -18,6 +18,9 @@ This documentation provides you with advanced configuration, detailed examples,
 !!! tip
     See the [FAQ][docs-faq] for some more tips!
 
+!!! important
```

Comment:
You may want to link to the new docs page about this that you're contributing. At the bottom of this document you can see we use reference links.

Internal docs follow the pattern of `docs-whatever` where the `docs-` prefix makes it more obvious it's internal (and easier to maintain via file search queries/filters), unlike other links, these should be relative file paths from the current documents location to the target documentation.

Perhaps append a mention of the relevant docs page "for more information".

Reply:
Good idea. I have implemented it.